app/assets/javascripts/modules/collapsible.js

```diff
@@ -5,26 +5,35 @@
   function Collapsible(section){
     this.$section = section;
     this.$clickTarget = this.$section.find('.js-subsection-title');
-    this.$clickTarget.on('click', this.toggle.bind(this));
+    this.$clickTarget.on('click', this.handleClick.bind(this));
     this.addToggle();
 
     this.$section.on('focus', '.js-subsection-body a', this.showSectionWhenLinkFocused.bind(this));
   }
 
+  Collapsible.prototype.handleClick = function handleClick(){
+    this.toggle()
+    this.toggleAttribute()
+  };
+
+  Collapsible.prototype.toggleAttribute = function() {
+    var $button = $(this.$clickTarget[0].firstChild)
+    $button.attr('aria-expanded', ($button.attr('aria-expanded') === "false" ? true : false));
+  };
+
   Collapsible.prototype.showSectionWhenLinkFocused = function() {
     if (this.$section.is('.closed')) {
       this.$section.toggleClass('closed');
     }
   };
 
   Collapsible.prototype.addToggle = function addToggle(){
-    var $toggleHTML = $("<span class='js-toggle'></span>");
+    var $toggleHTML = $("<span class='js-toggle' aria-hidden='true'></span>");
```

Comment:
Could you explain why you've added `aria-hidden`? (And do so in the commit - I couldn't find an explanation...?)

Reply:
done in the commit